src/UserInterface/updateActors.js

```diff
@@ -0,0 +1,46 @@
+
+function updateSliceActors(store) {
```

Comment:
`updateSliceActors` -> `updateSliceProperties` for consistency with the other function and since it mostly updates the properties.